supertokens_python/recipe/session/asyncio/__init__.py

```diff
@@ -11,7 +11,7 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations
 # under the License.
-from typing import Any, Dict, List, Union
+from typing import Any, Dict, List, Union, TypeVar, Callable, Optional
```

Comment:
It does and I don't think that it's a good idea to keep on using `Union`, it's length and make the code harder to read. Can we do this: in `utils.py` import `Optional` if available otherwise creating using `Option = Union[T, None]`, where `T` is TypeVar, and share it everywhere else. It will work seamlessly.

Reply:
TODO

Reply:
Okay cool.

Reply:
I did some digging and found that Python >= 3.7 clearly supports `from typing import Optional`

I think you might have confused it with `Literal` which needs to be imported from `typing_extensions` for python 3.7

So this utils thing is not required.